website/docs/components/accordion/partials/code/how-to-use.md

```diff
@@ -101,12 +101,12 @@ With a link in the toggle block and a form in the content.
         </div>
       </:toggle>
       <:content>
-        <p>
+        <Doc::Layout @spacing="16px" @direction="vertical">
```

Comment:
This is a code snippet that people will copy & paste, we can't use `Doc` components here. We just have to leave the `<p>` and let it collapse with the button. It's an example, it's OK even if it's not perfect.

Reply:
Ah good point, will adjust.

Reply:
good point! I removed.